File: src/main/java/com/khlud/ciprian/tranj/classesmodel/Variable.java
package com.khlud.ciprian.tranj.classesmodel;

import com.github.javaparser.ast.expr.Expression;

/**
 * Created by cipriankhlud on 19/10/2016.
 */
public class Variable extends NamedVariable {

    public boolean isStatic;
    public Expression initialValue;
}

File: src/main/java/com/khlud/ciprian/tranj/classesmodel/variables/Variable.kt
package com.khlud.ciprian.tranj.classesmodel.variables

import com.github.javaparser.ast.expr.Expression
import com.khlud.ciprian.tranj.resolvers.ResolvedType

/**
 * Created by Ciprian on 10/31/2016.
 */
open class NamedVariable {

    var name = ""
    var type: ResolvedType? = null
}

class Variable : NamedVariable() {

    var isStatic: Boolean = false
    var initialValue: Expression? = null
}

class Argument : NamedVariable()
